src/Compilers/Core/Portable/StrongName/StrongNameKeys.cs

```diff
@@ -95,7 +95,6 @@ internal static StrongNameKeys Create(string keyFilePath, CommonMessageProvider
 
             try
             {
-                Debug.Assert(PathUtilities.IsAbsolute(keyFilePath));
```

Comment:
So, let's keep the assert then, because the proper fix is not to remove it, but to normalize the path before this API is called.